NGRAPH_README.md

```diff
@@ -154,6 +154,18 @@
              python example/image-classification/train_mnist.py
              ```
 
+## Distributed training
+MPI is required for multi-CPU support. Download Open MPI from [here](https://www.open-mpi.org/).
+
+`USE_NGRAPH_DISTRIBUTED` must be set to (exactly) `1` in order for MXNet to build with nGraph distributed support.
+
+Here's an example to run ResNet-50 on two nodes:
+
+``` sh
+export MXNET_NGRAPH_GLUON=1
+mpirun -map-by node -x MXNET_NGRAPH_GLUON -x LD_LIBRARY_PATH -hostfile hosts -np 2 python MXNET_ROOT/example/image-classification/train_cifar10.py --network resnet --num-layers 50 --kv-store ngraph
```

Comment:
does users need to do any preparation like copying code/data to all nodes , into same dir structure.

Reply:
Either copying the data or using NFS would work. We populate the number of workers in the run and then rely on the data iterator to split the data across nodes.